src/components/Tooltip/tooltip.module.scss

```diff
@@ -4,16 +4,24 @@ $tooltip-arrow-shadow: 0px 1px 2px rgba(15, 20, 31, 0.12);
   cursor: pointer;
   display: inline-block;
 
-  &.trigger-above {
+  // Ensure portaled tooltip triggers are above when triggerAbove.
+  &.trigger-above[aria-expanded='true'] {
     z-index: $z-index-500;
+
+    &.tooltip {
+      z-index: $z-index-400;
```

Comment:
comment for future consideration... i find it easier to reason about zindex values if they are named according to what they representing. then when you have the definitions lined up against each other you can decided if they make sense

something like:
$z-index-tooltip: 400;
$z-index-modal-backdrop: 499;
$z-index-modal: 500;
etc